bigtable_automation/gcf/proto/dc_manifest.proto

```diff
@@ -0,0 +1,160 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Note: This file is copied from datacommons/proto/dc_manifest.proto
+// Please do not modify this file directly.
```

Comment:
Make this an IMPORTANT NOTE:?  Also add it to the g3 version (maybe a TODO for presubmit alert too)

Reply:
Improved the comment, will follow up in g3 version for comment.